### DISCUSSION TITLE
HTTP response status code for update and delete method?

### DISCUSSION THREAD

**Discussion by muath-ye** [laravel/framework#45796] [General]
Hi Community

I am building an API endpoints and following RESTful architectural style.
but I don't now which response status code perfect for each endpoint
Can you help me and the community to build perfect endpoints 

please support your answers with references so we can follow up .

Thanks

**Comment by denizgolbas** [comment]
https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete
